functions.js

```diff
@@ -294,7 +294,29 @@ exports.providerMessageHandler = (req, res) => {
 
     return kp.sendMsg(req, res);
 };
+//Eksaath Callback from StepOne For Workflow stages
+exports.providerEksaathHandler = (req,res) => {
+    //Authenticate
+    if (!req.query || req.query.api_key != config.eksaath_callback.api_key.toString()) {
+        return res.status(403).send({ errorMessage: "Unauthorized Access" });
+    } else {
+        delete req.query.api_key;
+    }
+    console.log("Query: ", JSON.stringify(req.query));
```

Comment:
Do not dump whole objects. Makes it impossible to debug anything. Put in one or two specific parameters if you want

Reply:
Changed this to a POST call